JavaAssignment/src/shashank/NullPointerException_Q1Feb16.java

```diff
@@ -0,0 +1,23 @@
+package shashank;
+
+public class NullPointerException_Q1Feb16 
+{
+	public static void main(String[] args) {
+		NullPointerException_Q1Feb16 np =  new NullPointerException_Q1Feb16();
+		np=null;
+		try{
+		np.m1();
+		}
+		catch(NullPointerException n){
+			NullPointerException_Q1Feb16 np1 =  new NullPointerException_Q1Feb16();
+			np=np1;
+			np.m1();
```

Comment:
Here you need to print that Null Pointer Exception Comes
but its makes no sense when user calling on null point even you get output. it's logically wrong.
please print some thing in catch block so user got he's calling on null point.